### ISSUE TITLE
Installation of Bio::Tree::Tree fails under perl-5.30.3

### ISSUE THREAD

**Issue by MartGro** [bioperl/bioperl-live#344]
I was trying to install Bio::Tree::Tree via cpanm under perl-5.30.3, but the installation fails.
The error message is:
-> FAIL Installing XML::DOM::XPath failed. See /home/stud/ga42vor/.cpanm/work/1591284525.30081/build.log for details. Retry with --force to force install it.
-> FAIL Installing the dependencies failed: Module 'DB_File' is not installed, Module 'XML::DOM::XPath' is not installed
-> FAIL Bailing out the installation for BioPerl-1.7.7.

Here are the log files: 
[https://gist.github.com/MartGro/9ad44ac4c27aed002ab94f54fbbd3ca7#file-build-log](https://gist.github.com/MartGro/9ad44ac4c27aed002ab94f54fbbd3ca7#file-build-log)
[https://gist.github.com/MartGro/c3a75c59b2ad9fe0b955153e49931496#file-build-log](https://gist.github.com/MartGro/c3a75c59b2ad9fe0b955153e49931496#file-build-log)

Do you need additional information? I'm an absolute beginner when it comes to perl and I only need to run one program. 
The installation instructions of that program indicate that it was built with BioPerl-1.5.2. 
If it does not work to get version 1.7.7 to run, is there a way to go back to version 1.5.2?

Thanks a lot, 
Martin

**Comment by MartGro** [comment]
I just saw that this is probably the same underlying issue as here:
[https://github.com/bioperl/bioperl-live/issues/336](https://github.com/bioperl/bioperl-live/issues/336)

All recent builds for the XML library are failing. Is there a way of working around that? Maybe including a older version (0.13 instead of 0.14?)

**Comment by ocstx** [comment]
In my case was due to the fact that XML::DOM::XPath is outdated.
Here explanation and solution:
https://superuser.com/questions/1427147/mac-os-x-mojave-perl-5-xmldomxpath-test-non-ascii-fail
but this is a "chapuza", I believe that a correction in "XML::DOM::XPath" should be applied
I'm using perl-5.30.1 installed with perlbrew in a High sierra OSX

**Comment by cjfields** [comment]
Yes this will likely need a fix in the upstream module or a change in the XML dependency (standardizing on XML::LibXML would be my suggestion).
